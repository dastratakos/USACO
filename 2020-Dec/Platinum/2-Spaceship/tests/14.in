20 20 20
11101111111111110101
11111011010101011101
11101111101111110111
11011111111110111100
10010010111101111011
11010011111101101010
01011101101001001010
11010111110011111010
10101111010100110110
11110010111111111011
11111111110010111011
10011101110111010011
11111110011010111111
10100011000101111001
01011110111000111111
11110111111001111111
11011101001101111111
11111111100011111111
11110111110010110001
11010110110111101111
3 18 4 13
5 14 4 12
1 13 3 15
5 17 3 13
1 12 3 3
4 2 2 3
8 5 4 3
19 13 5 11
3 20 5 12
11 6 2 2
1 6 1 3
2 1 14 5
4 2 18 15
12 14 1 17
1 2 2 2
9 8 1 3
4 13 1 3
4 10 4 8
8 9 1 6
4 20 4 20
