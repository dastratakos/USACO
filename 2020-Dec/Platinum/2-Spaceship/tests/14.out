385390419
454311707
699513282
332500375
973346913
202133205
949948294
952006502
11953672
344557108
300186523
972824039
960322308
426309157
870549091
949918805
2548834
705646652
458990257
690122077
